galpy/potential/interppotential_c_ext/interppotential_calc_potential.c

```diff
@@ -17,6 +17,7 @@
 #include <actionAngle.h>
 #include <integrateFullOrbit.h>
 #include <interp_2d.h>
+#include <sf_math.h>
```

Comment:
Doesn't seem necessary here (there is no other change in this file, so can't be that anything from this is now used).

Reply:
deleted.

Reply:
readied for test.

Reply:
So this is necessary? Seems weird if it's not used...

Reply:
not sure. Have to test without it. I just re-added many of the things I deleted when it stopped working.